### PR TITLE
Login Screen

### PR DESCRIPTION
**Pull request by AhmedElmasry700**

### CODE REVIEW

**Review by abualgait — COMMENTED** [review]
Great job Ahmed.
https://drive.google.com/file/d/1PsUUMJmneGOzw3ZnEZe7t00k4mTUCgc8/view?usp=sharing

Find assets here and create new pull request and try to :
1- create a style for every view so that we can use this style anywhere else in the code 
2 - make the ids naming in camelCase

File: app/src/main/java/com/resala/mobile/qrregister/ui/navhostactivity/NavHostActivity.kt [hunk 1]
```diff
@@ -13,10 +15,13 @@ import org.koin.android.viewmodel.ext.android.viewModel
 class NavHostActivity : BaseActivity<NavHostActivityVm>() {
 
     override val vm: NavHostActivityVm by viewModel()
-    override var layoutId: Int = R.layout.activity_nav_host
+    override var layoutId: Int = R.layout.login_frag
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(layoutId)
+    }
```
Comment (abualgait) [inline app/src/main/java/com/resala/mobile/qrregister/ui/navhostactivity/NavHostActivity.kt]: no need for these lines if you just want to set content view 
` override var layoutId: Int = R.layout.login_frag` do the job of setting content view

File: app/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
+    <style name="AppTheme" parent="Theme.MaterialComponents.Light.NoActionBar.Bridge">
```
Comment (abualgait) [inline app/src/main/res/values/styles.xml:9]: is this for DarkMode Support ?

**Comment by abualgait** [comment]
`NavHostActivity` supposed to be for Navigation component hosting frags

File: app/src/main/res/values/styles.xml (hunk 2 above)
Comment (AhmedElmasry700) [reply app/src/main/res/values/styles.xml:9]: no, this for material design EditText

Comment (abualgait) [reply app/src/main/res/values/styles.xml:9]: Ok great.

**Pull request merged**